en/docs/learn/identifier-first-flow-handler.md

```diff
@@ -0,0 +1,91 @@
+
+## Configuring Identifier-first handler in the login flow
+
+!!! warning
+    This handler can be configured at any step in the authentication flow. However, it is not an authenticator by itself and needs to be configured along with another authenticator in order for the authentication process to be successful.
+
+1.  Make sure that the validate_username parameter in the IS_HOME/repository/conf/deployment.toml file is set to true if you want your username validated first.
+
+    ```toml
+    [authentication.authenticator.user_identifier] 
+    name ="IdentifierExecutor"
+    enable=true
+    [authentication.authenticator.user_identifier.parameters]
+    validate_username= true
+    ```
+2.  Log in to the WSO2 IS [management console](../../setup/getting-started-with-the-management-console/). Select a Service Provider (For details on how to create a new service provider, click [here](../../learn/adding-and-configuring-a-service-provider)) and expand the **Local & Outbound Authentication Configuration** section. Click **Advanced Configuration** and add the identifier-first as one of the steps as shown below. Since this handler does not work as an independant authenticator, you have to add another authentication step with the basic authenticator to save this configuration. 
+
+![identifier-first](../assets/img/learn/identifier-first.png)
+
+## Let's try this out for a scenario!
+
+Suppose the admin wants the user who attempts to log in, to be authenticated by a federated authenticator that is chosen based on the domain name specified. Let's also assume that the user has opted to validate the identity before proceeding to authenticate the user. 
+
+Follow the steps given below to accomplish this. 
+
+!!! tip "Before you begin"
+    
+    1.  Install WSO2 IS 5.9.0 by downloading the [installer](https://wso2.com/identity-and-access-management/)
+
+    2.  Navigate to <IS_HOME>/repository/conf/deployment.toml file. Set the enable_email_domain configuration to true to enable email authentication. By doing this, we are configuring WSO2 IS to use the email as the username.
+
+        ```toml
+        [tenant_mgt]
+        enable_email_domain= true
+        ```
+
+    3.  [Download](https://tomcat.apache.org/download-80.cgi) and install Apache Tomcat version 8.*.* or above.
+
+    4.  Open the /etc/hosts file and add the following entry.
+
+        ```bash
+        127.0.0.1 localhost.com
+        ```
+    5.  Restart WSO2 Identity Server.
+
+1.  Download the saml2-web-app-pickup-dispatch.com.war file from [here](https://github.com/wso2/samples-is/releases) and copy it inside the <TOMCAT_HOME>/webapps directory.
+
+2.  Start the tomcat server.  
+
+3.  In the WSO2 IS [management console](https://localhost:9443/carbon/admin/login.jsp), create a new service provider and expand the Inbound Authentication configuration. Select the SAML2 Web SSO configuration section, and click Configure.
```

Comment:
Bold names of buttons or expands such as **Inbound Authentication Configuration** and **Configure**.


Instead of saying "**the** Inbound Authentication configuration", shall we cut off "the" or say "the Inbound Authentication configuration **section**"